spec/models/cloud_service_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe CloudService, :type => :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
I would think you should just remove this file altogether, since we don't need more pending things.